README.md

```diff
@@ -580,13 +584,19 @@ func getBookable(c *gin.Context) {
 ```
 
 ```console
-$ curl "localhost:8085/bookable?check_in=2017-08-16&check_out=2017-08-17"
+$ date +'%m/%d/%Y'
+03/09/2018
```

Comment:
remove the command?

Reply:
I can merge this PR after removing above command.

Reply:
Done.